docs/using-the-nodejs-wrapper/SessionState.md

```diff
@@ -0,0 +1,42 @@
+# Session States
+
+## What is a session state?
+
+Every connection is associated with a connection session on the server and a group of related session settings like the autoCommit flag or the transaction isolation level. The following session settings are tracked by the AWS NodeJS Driver and together they form a session state:
```

Comment:
Please replace all instances of `AWS NodeJS Driver` with `AWS Advanced NodeJS Wrapper`